### ISSUE TITLE
Some rough ETA?

### ISSUE THREAD

**Issue by oscarbg** [gongminmin/Dilithium#5]
I know.. I know.. project seems still in it's infancy..
but can provide some rough ETA? for example do you think project (at least DXIL->SPIR-V) can be more or less implemented on 2017?
also note I asked support for DXIL<->SPIR-V here: https://github.com/Microsoft/DirectXShaderCompiler/issues/68
and now pointed to your project so you get some publicity :-)

**Comment by gongminmin** [comment]
Thanks! I'm pretty close to make the first step: disasm of DXIL. I hope I can push it this weekend or early next week. Then I'll move on to SPIR-V generator. It definitely can be implemented in 2017, very likely before September.

**Comment by oscarbg** [comment]
Nice news.. will follow the project frequently for updates..

**Comment by oscarbg** [comment]
have you lost interest on it? no pressure just haven't seen any progress in a while..

**Comment by gongminmin** [comment]
Well, there are google engineers involve in that open source D3D compiler, to add a SPIR-V back-end. So I slowed this down. I think the goal of Dilithium is now provide a lean-and-mean binary translator.

**Comment by oscarbg** [comment]
Yes that's should be the plan IMHO.. must be of use to users likes Wine
developers once games using DXIL binaries instead of DXBC emerge and they
want to translate to SPIR-V

2017-07-29 7:16 GMT+02:00 Minmin Gong <notifications@github.com>:
